packages/client/hmi-client/src/components/Navbar.vue

```diff
@@ -96,14 +99,18 @@ function updateProjectNavItem(id, name) {
 	const projectNavItem = {
 		[projectNavKey]: {
 			name,
-			icon: 'pi pi-images',
+			icon: 'pi pi-clone',
 			routeName: RouteName.ProjectRoute
 		}
 	};
 	navItems.value = { ...initialNavItems, ...projectNavItem };
 	selectedPage.value = navItems.value[currentRoute.value.path];
 }
 
+function closeLogoutDialog() {
+	isLogoutDialog.value = false;
```

Comment:
In my opinion we should do this assignment on the cancel button's `onClick` since it is only one line and only used once.